source/NVDAObjects/UIA/winConsoleUIA.py

```diff
@@ -215,6 +235,8 @@ class WinConsoleUIA(Terminal):
 	#: Whether the console got new text lines in its last update.
 	#: Used to determine if typed character/word buffers should be flushed.
 	_hasNewLines = False
+	#: the caret in consoles can take a while to move on Windows 10 1903 and later.
+	_caretMovementTimeout = 2
```

Comment:
I think you mean _caretMovementTimeoutMultiplier?

Reply:
Yes, yes I did.